ui/src/components/card/CardInstance.tsx

```diff
@@ -157,13 +180,22 @@ const CardInstance = (props: {
             <InstanceNumber>#{props.instanceNumber}</InstanceNumber>
           </InstanceName>
 
-          {!isOverlayVisible && (
-            <SmallStatusIcon
+          {!isStateOverlayVisible && (
+            <SmallStateIcon
+              title="Show Info"
+              onClick={() => setIsStateOverlayVisible(true)}
+            >
+              {getStateIcon(stateCode)}
+            </SmallStateIcon>
+          )}
+
+          {!isStateOverlayVisible && (
+            <SmallHealthIcon
               title="Show Info"
-              onClick={() => setIsOverlayVisible(true)}
+              onClick={() => setIsHealthOverlayVisible(true)}
             >
-              {getStatusIcon(statusCode)}
-            </SmallStatusIcon>
+              {getHealthIcon(healthCode)}
+            </SmallHealthIcon>
           )}
```

Comment:
Shall we just remove the checks here? I think the icons should show behind the overlay (now that we could have two showing).

Reply:
Yeah, good point - I agree